class.jetpack.php

```diff
@@ -4082,7 +4082,7 @@ function admin_page_load() {
 			$this->error = __( 'Cheatin&#8217; uh?', 'jetpack' );
 			break;
 		case 'access_denied' :
-			$this->error = __( 'You need to authorize the Jetpack connection between your site and WordPress.com to enable the awesome features.', 'jetpack' );
+			$this->error = __( 'Would you mind telling us why you did not complete the Jetpack connection in this <a href="http://surveys.jetpack.me/cancelled-connection">1 question survey</a><br><small>A Jetpack connection is required for our free Security, and Traffic features to work</small>.', 'jetpack' );
```

Comment:
What do you think about splitting the 2 sentences into 2 different strings? This you could move the `br` and `small` tag out of the string, and it would be easier to translate. Since the 2 sentences are not related, I think it's ok to split them.